app.json

```diff
@@ -4,13 +4,9 @@
     "slug": "dccentralkitchen-clerks",
     "privacy": "public",
     "sdkVersion": "35.0.0",
-    "platforms": [
-      "ios",
-      "android",
-      "web"
-    ],
+    "platforms": ["ios", "android", "web"],
     "version": "1.0.0",
-    "orientation": "portrait",
+    "orientation": "landscape",
```

Comment:
Didn't know this was controlled here, woah

Reply:
it seems to work on my phone so hopefully it's the same for tablets!!